rust/src/utils.rs

```diff
@@ -453,7 +453,8 @@ pub(crate) fn get_rpm_basearch() -> String {
     match std::env::consts::ARCH {
         "arm" => "armhfp".to_string(),
         "powerpc" => "ppc".to_string(),
-        "powerpc64" => "ppc64".to_string(),
+        "powerpc64" if cfg!(target_endian = "big") => "ppc64".to_string(),
+        "powerpc64" if cfg!(target_endian = "little") => "ppc64le".to_string(),
```

Comment:
LGTM as is, though... how much should we care about ppc64 big endian? Neither Fedora nor RHEL build for it, and my understanding is that it's generally falling out of support a bit everywhere.

I.e. should we just make this

```suggestion
        "powerpc64" => "ppc64le".to_string(),
```

Obviously, it's fine to carry, though that `"ppc64".to_string()` may actually never have the privilege to be executed... ever. And it's unlikely we'll remember to care about that arch also in other places.

Reply:
Not at all, although I think that it doesn't hurt to have it there, if some one crazy enough would be around to use it there, like sparc :).